static/js/composebox_typeahead.js

```diff
@@ -442,6 +442,7 @@ export function filter_and_sort_mentions(is_silent, query, opts) {
 
 export function get_pm_people(query) {
     const opts = {
+        ...get_stream_topic_data(this),
```

Comment:
Have you successfully tested this change manually? I wouldn't expect `this` in this context to be set in a way that causes this to work.

Reply:
Yes, this works for me on the development server

Reply:
Can you talk about the reasoning for what you expect `this` to be here and why you expect that to produce the right result?

Reply:
Here's what I get from `console.log(this)` when testing in the browser:
![image](https://user-images.githubusercontent.com/453486/182698467-5a6450c2-2a86-4804-a03b-0ea8923700df.png)
However, looking at the implementation of `get_stream_topic_data`, I realized I can get the context from `compose_state` instead. I changed that line of code, and now the tests pass. I still need to add a test to prevent regressions.